### ISSUE TITLE
[BUG] Uncaught TypeError: Cannot read properties of undefined (reading 'ClientMsgCode') when using this modules with Parcel

### ISSUE THREAD

**Issue by nishidayoshikatsu** [liveblocks/liveblocks#537]
**Describe the bug**

When using this modules with Parcel, I get the error `Uncaught TypeError: Cannot read properties of undefined (reading 'ClientMsgCode')`.

**To Reproduce**

1. `import { createClient } from "@liveblocks/client";`
2. `createClient` is called

**Expected behavior**

Liveblocks app should work with parcel app.

**Environment (please complete the following information):**

- @liveblocks/react@0.18.5 @liveblocks/client@0.18.5
- chrome v106.0.5249.119

**Comment by nvie** [comment]
Thanks @nishidayoshikatsu for reporting. I can confirm that this indeed breaks on a standard Parcel build. There is a reason we generate the ESM build output the way we do, but apparently this format breaks when used with Parcel. We're going to be looking into the root cause of this this week, and see if we can find a way that works in all bundlers.

**Comment by nishidayoshikatsu** [comment]
Thanks @nvie, looking forward to it!

**Comment by ajwhite** [comment]
Same issue on codesandbox.io, as seen here: https://codesandbox.io/s/github/liveblocks/liveblocks/tree/main/examples/react-whiteboard?file=/src/liveblocks.config.js

**Comment by nvie** [comment]
Hey @nishidayoshikatsu, thank you so much for bringing this issue to our attention, and for the patience while we worked on a fix for this! 🙏

We've published a short term fix by removing ESM builds from our bundle, and temporarily only publishing CJS. This should make it work in all environments again. We do plan on publishing ESM modules again later, but only once we have a way to ensure it works in the entire bundler landscape.

If you upgrade to ✨ **0.19.1** ✨ , these import issues should disappear!

cc @odiak @atticoos
